admin/new.php

```diff
@@ -428,7 +433,14 @@ function gigpress_add() {
 					<td><input type="text" size="48" name="venue_address" id="venue_address" value="<?php if(isset($venue_address)) echo $venue_address; ?>" /></td>
 				</tr>
 				<tr>
-					<th scope="row"><label for="venue_city"><?php _e("Venue city", "gigpress") ?>:<span class="gp-required">*</span></label></th>
+					<th scope="row">
+						<label for="venue_city">
+							<?php _e("Venue city", "gigpress") ?>:
+							<?php if ( true === $venue_city_required ) { ?>
+								<span class="gp-required" >*</span >
```

Comment:
```suggestion
								<span class="gp-required" >*</span>
```
I think the white space will break the HTML, and the closing span won't work.